dapp/src/components/wizard/pages/SummaryStep.jsx

```diff
@@ -1,20 +1,45 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 import FontAwesome from 'react-fontawesome'
-import CommunityLogo from 'components/common/CommunityLogo'
+// import CommunityLogo from 'components/common/CommunityLogo'
 import { connect, getIn } from 'formik'
+import useSwitchNetwork from 'hooks/useSwitchNetwork'
+import pickBy from 'lodash/pickBy'
+import pluginsIcons from 'constants/pluginsIcons'
+import upperCase from 'lodash/upperCase'
+import lowerCase from 'lodash/lowerCase'
+import upperFirst from 'lodash/upperFirst'
 
 const SummaryStep = ({
   networkType,
-  formik
+  formik,
+  homeNetwork
 }) => {
+  const desiredNetworkType = useMemo(() => {
+    if (!networkType) {
+      return ['mainnet', 'ropsten']
```

Comment:
why `['mainnet', 'ropsten']` and then `['ropsten', 'mainnet']`?

Reply:
wasn't relevant